modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-okhttp/api.mustache

```diff
@@ -11,6 +11,7 @@ import kotlinx.coroutines.withContext
 {{/useCoroutines}}
 {{/doNotUseRxAndCoroutines}}
 import {{packageName}}.infrastructure.ApiClient
+import {{packageName}}.infrastructure.ApiInfrastructureResponse
```

Comment:
I'll rename ApiInfrastructureResponse to ApiResponse in another PR later to make it consistent with other clients (e.g. Java, C#, etc)

Reply:
Can you please create a typealias to make the transition easier?
```
@Deprecated("Renamed to ApiResponse", ReplaceWith("ApiResponse"))
typealias ApiInfrastructureResponse = ApiResponse
```